src/index.js

```diff
@@ -1,2 +1,3 @@
-import BoilerplateComponent from "./components/boilerplate-component";
-export default BoilerplateComponent;
+module.exports = {
```

Comment:
My sense is for a common index is do named exports. And I think "old school" (`module.exports`) is fine since it's terse and easy to understand here...